content/news/2024.md

```diff
@@ -53,6 +53,15 @@ More info...
 - bug fix #2 
 -->
 
+## December 2024
+
+### <span class="label label-news label-added">ADDED</span> Added a New "**Product Updated Delta**" Event For Event Platform
+
+This new "**product.updated.delta**" event contains the product delta, allowing you to see the previous and new values of your products.
+
+👉 Learn more about [this new event ](/event-platform/available-events.html#comakeneopimv1productupdateddelta)
+
+
```

Comment:
Text to improve! (if it's relevant to add something in this section)
If you have any idea, cc. @ThomasBigot @tomglvng